### PR TITLE
Update custom-job.yaml

### PR DESCRIPTION
**Pull request by galipremsagar**

### CODE REVIEW

File: .github/workflows/custom-job.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 on:
   workflow_call:
+    outputs:
+      job_output:
+        description: "job_output set by inputs.run_script"
+        value: "${{ jobs.build.outputs.job_output }}"
```
Comment (galipremsagar) [inline .github/workflows/custom-job.yaml:6]:
```suggestion
```

File: .github/workflows/custom-job.yaml [hunk 2]
```diff
@@ -46,6 +50,11 @@ permissions:
 
 jobs:
   build:
+    outputs:
+      # Export run_step_output to dependent jobs
+      # To set this, the script must do
+      # echo "job_output=WHATEVER" >> "${GITHUB_OUTPUT}"
+      job_output: ${{ steps.run-step.outputs.job_output }}
```
Comment (galipremsagar) [inline .github/workflows/custom-job.yaml:57]:
```suggestion
```

**Comment by bdice** [comment]
@galipremsagar Is this still needed or can it be closed?

**Comment by galipremsagar** [comment]
> @galipremsagar Is this still needed or can it be closed?

It can be closed.

**Pull request closed without merging**